controllers/components/modelregistry/modelregistry_support.go

```diff
@@ -0,0 +1,119 @@
+package modelregistry
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+	"text/template"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	componentsv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1"
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
+	infrav1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/infrastructure/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
+	odhtypes "github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/conversion"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+
+	_ "embed"
+)
+
+const (
+	ComponentName                   = componentsv1.ModelRegistryComponentName
+	DefaultModelRegistriesNamespace = "odh-model-registries"
+	DefaultModelRegistryCert        = "default-modelregistry-cert"
+	BaseManifestsSourcePath         = "overlays/odh"
+)
+
+var (
+	imagesMap = map[string]string{
+		"IMAGES_MODELREGISTRY_OPERATOR": "RELATED_IMAGE_ODH_MODEL_REGISTRY_OPERATOR_IMAGE",
+		"IMAGES_GRPC_SERVICE":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
+		"IMAGES_REST_SERVICE":           "RELATED_IMAGE_ODH_MODEL_REGISTRY_IMAGE",
+	}
+
+	extraParamsMap = map[string]string{
+		"DEFAULT_CERT": DefaultModelRegistryCert,
+	}
```

Comment:
Becase the same constant is used in other context and actions. But in general , I'd prefer to use constants instead of string so it is easy to spot where they are in use.